style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
+    padding-top: 8rem;
+    margin-left: 5rem;
+  }
+
+  #About .Aboutcontent h3 {
+    width: 100%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  #About .Aboutcontent p {
+    width: 447px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: #344563;
+    padding-top: 1.2rem;
+    padding-bottom: 1rem;
+  }
+
+  #About .Aboutcontent a {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    color: #7f8cff;
+    text-decoration: none;
+    margin-top: -0.2rem;
+  }
+
+  #About .iconsAbout ul {
+    display: flex;
+    list-style: none;
+  }
+
+  #About .iconsAbout a {
+    padding-left: 1.5rem;
+  }
+
+  #About .info {
+    margin-top: 9.5rem;
+    margin-right: 5rem;
+  }
+
+  #About .other h5 {
+    height: 1.5rem;
+    margin-left: 2.5rem;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    padding-top: 2.5rem;
+    padding-bottom: 2.5rem;
+    color: #344563;
+  }
+
+  #About .inline h5 {
+    height: 1.5rem;
+    color: #344563;
+    margin-left: 0.4rem;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 650;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+  }
+
+  #About .info .inline .arrow {
+    margin-top: 0.4rem;
+    margin-right: 2.3rem;
+    position: relative;
+    left: 90%;
+    bottom: 25px;
+  }
+
+  #About .languages ul {
+    display: flex;
+    list-style: none;
+  }
+
+  #About .languages .background {
+    width: 110px;
+    height: 7rem;
+    background: #f7f7f9;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+  }
+
+  #About .languages .background .e {
+    padding: 0.5rem;
+  }
+
+  #About .background a {
+    font-family: 'Poppins', sans-serif;
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    color: hsl(218, 41%, 25%);
+  }
+
+  #About .info .languages .js {
+    margin: 1rem;
+    position: inherit;
+  }
+
+  #About .info .languages .background .html,
+  #About .info .languages .background .css {
+    display: block;
+    position: inherit;
+    margin-left: -2.5rem;
+  }
+
+  #About .other {
+    position: relative;
+    top: 8rem;
+  }
+
+  #About .other hr {
+    width: 90%;
+    margin-left: 2.3rem;
+    background: #dfe1e6;
+  }
+
+  #About .other img {
+    margin-top: 0.4rem;
+    margin-right: 2.5rem;
+    position: relative;
+    left: 90%;
+    bottom: 37px;
+  }
+
+  footer #contact-form {
+    margin-top: -5rem;
+    display: grid;
+    background-image: url("Images/Footer_Desktop.png");
+    background-size: cover;
+    width: 100%;
+    height: auto;
+  }
```

Comment:
- [ ] Kindly add some spacing at the top of the contact section and inside the contact section in both mobile and desktop. You are expected to stick to the design template as much as possible. This includes margins and paddings.
### Your design
![image](https://user-images.githubusercontent.com/94551484/236448141-e2e50e81-2eaf-4782-8b72-154e58a5833d.png)
![image](https://user-images.githubusercontent.com/94551484/236448276-39da2436-f248-4a9d-8763-bb0c2ff759b7.png)


### Figma
![image](https://user-images.githubusercontent.com/94551484/236448228-3f673d25-c74f-4cb4-8fed-eaed2f22dbb9.png)
![image](https://user-images.githubusercontent.com/94551484/236448337-9179c877-cea9-4b4d-a9e9-eac120f53344.png)